src/SFA.DAS.FAT.Web/Views/SearchCourses/Index.cshtml

```diff
@@ -0,0 +1,75 @@
+﻿@using Microsoft.AspNetCore.Mvc.TagHelpers
+@using SFA.DAS.FAT.Web.Infrastructure
+@using SFA.DAS.FAT.Web.Models
+@model SFA.DAS.FAT.Web.Models.SearchCoursesViewModel
+@{
+    ViewData["Title"] = "Search for apprenticeship training courses and training providers";
+    ViewData["Description"] = "Search for apprenticeship training courses and training providers";
+}
+
+<div class="govuk-grid-row">
+    <div class="govuk-grid-column-three-quarters">
+        @* <partial name="_validationSummary" /> *@
```

Comment:
My guess is that this is no longer required?

Reply:
correct